data.yml

```diff
@@ -0,0 +1,27 @@
+---
+homepage-header: My Home Page
+registration_page_header: Registration form
+start-page-header: Hello 🌱, Do you have an account?
+records-list-page-title: Records List
+log-in-page-header: Log in
+create-person-page-title: Create a Person
+cities-statistics-page-title: Cities Statistics
+delete-person-record-page-title: Delete Person Record
+
+page-not-found-error: 404 - Page not found
+not-authorised-error: 401 error - Not Authorised
+record-access-error: 403 error - Action is forbidden
+
+successful-record-delete: Record deleted successfully
+
+invalid-username-or-password-error: Invalid username or password!
+username-taken-error: Username has already been taken
+blank-password-error: Password can't be blank
+blank-first-name-error: First name can't be blank
+blank-second-name-error: Second name can't be blank
+blank-username-error: Username can't be blank
+username-length-error: Username username length should be 4-20 symbols
+password-length-error: Password password must have length from 6 to 15 sym.
+blank-password-confirmation-error: Password confirmation can't be blank
+blank-city-error: City can't be blank
+city-length-error: City length should be between 1 and 100 symbols
```

Comment:
As usual, we use underscore for variable 
`city_length_error:`

Reply:
got it 😊